test/NinjaTools.FluentMockServer.API.Exclude.Tests/Downstream/RequestMapperSpec.cs

```diff
@@ -63,10 +63,7 @@ public RequestMapperSpec(ITestOutputHelper outputHelper) : base(outputHelper)
             {
                 actual.Properties.Should().NotBeNull();
                 actual.Properties.ToList().Count.Should().Be(expected.Properties.ToList().Count);
-                if (expected.Properties.Any())
-                {
-                    actual.Properties.ToList().Should().Contain(expected.Properties.ToList());
-                }
+                if (expected.Properties.Any()) actual.Properties.ToList().Should().Contain(expected.Properties.ToList());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4739715)